src/arkode/arkode_arkstep_io.c

```diff
@@ -1290,132 +1290,63 @@ int arkStep_PrintAllStats(ARKodeMem ark_mem, FILE* outfile, SUNOutputFormat fmt)
   retval = arkStep_AccessStepMem(ark_mem, __func__, &step_mem);
   if (retval != ARK_SUCCESS) { return (retval); }
 
-  switch (fmt)
+  /* function evaluations */
+  sunfprintf_long(outfile, fmt, SUNFALSE, "Explicit RHS fn evals", step_mem->nfe);
```

Comment:
I like how the new `sunfprintf` functions really tighten things up with the table-vs-csv output, but it would be nice if these were documented somewhere (e.g., in the developer guide), since it took me a while to discover their definitions inside `src/sundials/sundials_utils.h`.

Reply:
Also, have the outputs from these been compared with the previous outputs (especially for the CSV)?

Reply:
> I like how the new sunfprintf functions really tighten things up with the table-vs-csv output

Credit to @gardner48 from an earlier incarnation of this PR

> it would be nice if these were documented somewhere

Agreed

> Also, have the outputs from these been compared with the previous outputs (especially for the CSV)?

I haven't. Looks like there's some test that use CSV output, so I'll check those diffs.

Reply:
> > it would be nice if these were documented somewhere
> 
> Agreed

The documentation could go in [this section](https://sundials.readthedocs.io/en/develop/developers/source_code/Style.html) of the developer guide.

> > Also, have the outputs from these been compared with the previous outputs (especially for the CSV)?
> 
> I haven't. Looks like there's some test that use CSV output, so I'll check those diffs.

I checked this manually in the initial implementation but we should add a unit test to catch any regressions